ocs_ci/utility/lvm_utils.py

```diff
@@ -30,64 +30,82 @@ def osd_scenario1(osd, devices_dict, dmcrypt=False):
+        osd,
+        osd.LvmConfig.data_lv %
+        '1',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(20)
+    )
+    data_lv2 = lvcreate(
+        osd,
+        osd.LvmConfig.data_lv %
+        '2',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(20)
+    )
+    data_lv3 = lvcreate(
+        osd,
+        osd.LvmConfig.data_lv %
+        '3',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(20)
+    )
+    data_lv4 = lvcreate(
+        osd,
+        osd.LvmConfig.data_lv %
+        '4',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(20)
+    )
     db_lv1 = lvcreate(
         osd,
         osd.LvmConfig.db_lv %
         '1',
         osd.LvmConfig.vg_name %
         '1',
-        osd.LvmConfig.size.format(8))
+        osd.LvmConfig.size.format(8)
+    )
     db_lv2 = lvcreate(
         osd,
         osd.LvmConfig.db_lv %
         '2',
         osd.LvmConfig.vg_name %
         '1',
-        osd.LvmConfig.size.format(8))
-    wal_lv1 = lvcreate(osd,
-                       osd.LvmConfig.wal_lv %
-                       '1',
-                       osd.LvmConfig.vg_name %
-                       '1',
-                       osd.LvmConfig.size.format(2))
-    wal_lv2 = lvcreate(osd,
-                       osd.LvmConfig.wal_lv %
-                       '2',
-                       osd.LvmConfig.vg_name %
-                       '1',
-                       osd.LvmConfig.size.format(2))
-
-    scenario = "{{'data':'{datalv1}','data_vg':'{vg_name}'}},{{'data':'{datalv2}','data_vg':'{vg_name}'," \
-               "'db':'{dblv1}','db_vg':'{vg_name}'}}," \
-               "{{'data':'{datalv3}','data_vg':'{vg_name}','wal':'{wallv1}','wal_vg':'{vg_name}'}}," \
-               "{{'data':'{datalv4}','data_vg':'{vg_name}','db':'{dblv2}','db_vg':'{vg_name}','wal':'{wallv2}'," \
-               "'wal_vg':'{vg_name}'}}".format(vg_name=vgname, datalv1=data_lv1, datalv2=data_lv2, dblv1=db_lv1,
-                                               datalv3=data_lv3, wallv1=wal_lv1, datalv4=data_lv4,
-                                               dblv2=db_lv2, wallv2=wal_lv2)
+        osd.LvmConfig.size.format(8)
+    )
+    wal_lv1 = lvcreate(
+        osd,
+        osd.LvmConfig.wal_lv %
+        '1',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(2)
+    )
+    wal_lv2 = lvcreate(
+        osd,
+        osd.LvmConfig.wal_lv %
+        '2',
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(2)
+    )
+
+    scenario = (
+        "{{'data':'{datalv1}','data_vg':'{vg_name}'}},{{'data':'{datalv2}','data_vg':'{vg_name}',"
+        "'db':'{dblv1}','db_vg':'{vg_name}'}},"
+        "{{'data':'{datalv3}','data_vg':'{vg_name}','wal':'{wallv1}','wal_vg':'{vg_name}'}},"
+        "{{'data':'{datalv4}','data_vg':'{vg_name}','db':'{dblv2}','db_vg':'{vg_name}','wal':'{wallv2}',"
```

Comment:
I see this line is still quite long, can we maybe break this line as well?

Reply:
Black will organize the `.formats()` much better but not the strings if they are too long. I will eventually cover the long strings in subsequent PRs.